executors/cpp/main.cpp

```diff
@@ -83,44 +77,41 @@ int main(int argc, const char** argv)
       // TODO: get from the array of supported tests
       json_object *tests_supported = json_object_new_object();
       json_object *test_array = json_object_new_array();
-      for (int index = 0; index < 4; index ++) {
-        json_object_array_add(test_array,
-                              json_object_new_string(supported_tests[index].c_str()));
+      for (int index = 0; index < 7; index ++) {
```

Comment:
replace magic number with built in functions that you can combine for the length of an array:
https://stackoverflow.com/a/4108340/2077918

based on the other answers on the page, it looks like there are nicer versions for if we were using C++17 or C++20

Reply:
Hmm, I thought I had replaced this with a size call. Will revisit!